src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,32 @@
-export const Tabs = () => {};
+import classNames from 'classnames/bind';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
```

Comment:
The prop `selectedTabId` should be renamed to `activeTabId` to align with the task description. This change will ensure consistency across the codebase.